guides/_subsections/add-a-trusted-certificate.md

```diff
@@ -0,0 +1,33 @@
+## [Add a Trusted Certificate](#-add-a-trusted-certificate)
+If you have Java Management Extensions (JMX) over SSL enabled on your containerized JVMs, you must configure Cryostat to trust the SSL certificate presented by the containerized JVM when Cryostat attempts to open a JMX connection. If this configuration is not completed then Cryostat will be unable to open the connection and unable to perform JFR management tasks.
+
+Here's how to add a trusted SSL certificate with the Cryostat web UI.
+
+<ol>
+    <li>
+        {% include howto_step.html
+          summary="Navigate to the Security tab"
+          image-name="store-jmx-credentials-1.png"
+          text="
+            First, navigate to the <i>Security</i> tab. The Import SSL Certificates card contains an <i>Upload</i> button. Click <i>Upload</i> to open a file upload dialog and choose the certificate you wish to upload. You may repeat this process multiple times to add multiple trusted certificates.
```

Comment:
Break out to two steps and drop "first" (this nominal entry usually indicates more entries are to follow, such as "second, third, and so on)

1. Click the <i>Security</i> tab. 
2. Click the <i>Upload</i> button on the Import SSL Certificates card. This action opens a file-upload dialog, where you can choose the certificate that you want to upload to Cryostat. You can repeat this process multiple times to add multiple trusted certificates.